deprecated_store/index.ts

```
@@ -0,0 +1,396 @@
+//   network: Network | null | undefined;
+// };
+
+const SHOW_LOGS = false;
+
+const log = (...args: any[]) => {
+  if (SHOW_LOGS) {
+    console.log(...args);
+  }
+};
+
+type Query<T extends any = string, HasSet extends boolean = true> = {
```

Biome (1.9.4)

[error] Constraining a type parameter to any or unknown is useless.

All types are subtypes of any and unknown.
Safe fix: Remove the constraint.


(lint/complexity/noUselessTypeConstraint)